tests/wpt/tests/html/semantics/embedded-content/the-iframe-element/srcdoc-removed-iframe-crash.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <link rel="author" title="Martin Robinson" href="mrobinson@igalia.com">
```

Comment:
I'm pretty sure you're not mrobinson :)

Reply:
Apologies, this was sitting for 3 weeks so I "adopted" it to get it landed. 😅

Reply:
Oh, I totally missed your comment before this? My apologies!